tests/main/lxd-mount-units/task.yaml

```diff
@@ -0,0 +1,50 @@
+summary: Test mount units generated for snaps inside lxd container are correct.
+
+details: |
+    Test that mount units generated by snapd-generator inside lxd container
+    are correct for snapfuse.
+
+# only 20.10+, we want lxd images that come with snaps preinstalled.
+systems: [ubuntu-20.10*]
+
+execute: |
+    echo "Install lxd"
+    snap install lxd
+
+    lxd waitready
+    lxd init --auto
+
+    VERSION_ID="$(. /etc/os-release && echo "$VERSION_ID" )"
+    lxd.lxc launch --quiet "ubuntu:$VERSION_ID" ubuntu
+
+    lxd.lxc file push --quiet "$GOHOME"/snapd_*.deb "ubuntu/root/"
+
+    DEB=$(basename "$GOHOME"/snapd_*.deb)
+    lxd.lxc exec ubuntu -- apt install -y /root/"$DEB"
```

Comment:
is sudo needed here?

Reply:
No, it's not, lxc commands are run as root.